contracts/utils/Address.sol

```diff
@@ -15,6 +15,12 @@ library Address {
      * It is unsafe to assume that an address for which this function returns
      * false is an externally-owned account (EOA) and not a contract.
      *
+     * **You cannot rely on `isContract` to protect against flash loan attacks!**
+     * Using `isContract` to intentionally prevent calls from contracts should be avoided, as it:
+     * 1) breaks composability
+     * 2) breaks support for smart wallets like Gnosis Safe where every user is a contract
+     * 3) can be circumvented by calling from a contract constructor
```

Comment:
A comment about formatting: not sure if you meant this to be a numbered list in the docs site, but in the way it's written now it gets rendered inline in a paragraph ([see](https://deploy-preview-2994--openzeppelin-contracts-docs.netlify.app/contracts/4.x/api/utils#Address-isContract-address-)).

In order to make it a numbered list you need to add a newline after the colon and then use `1.` rather than `1)`.